report.py

```diff
@@ -1,11 +1,11 @@
-from datetime import datetime
+from datetime import date
 
 
 class Report:
 
     @staticmethod
-    def _convert_date(date):
-        return datetime.fromisoformat(date).strftime('%d.%m.%Y')
+    def _convert_date(date: date):
```

Comment:
it doesn't work well now - the visit time should be a just a time, but now we have a full date time in the report summary